static-builder/src/js/components/ui/button.js

```diff
@@ -1,13 +1,18 @@
 import { Teact } from '@/js/libs/teact'
 
-export const DefaultButton = ({ type = 'button', text, onClick }) => {
+export const DefaultButton = ({
+  type = 'button',
+  text,
+  onClick,
+  className = '',
+}) => {
   return Teact.createElement(
     'button',
     {
-      type: type,
-      className: 'btn btn-primary btn-lg bg-darkblue',
-      onClick: onClick,
+      type,
+      onClick,
+      className: `btn btn-primary btn-lg bg-darkblue ${className}`.trim(),
```

Comment:
```suggestion
      className: `btn btn-primary btn-lg bg-darkblue ${className}`,
```
試してみてないですが、これではダメそうです？